projects/vdk-plugins/vdk-kerberos-auth/src/vdk/plugin/kerberos/minikerberos_authenticator.py

```diff
@@ -75,7 +75,12 @@ def _kinit(self) -> None:
             async def get_tgt():
                 await krb_client.get_TGT()
 
-            asyncio.run(get_tgt())
+            # Create a separate event loop to avoid interfering with users'
+            # data jobs utilizing asyncio, as well.
```

Comment:
Explain in the code a bit more why we are doing this. Some future maintainer may wonder as well.

Reply:
Updated and expanded the comments. Thanks